contracts/Core/Parameters.sol

```diff
@@ -23,7 +23,9 @@ contract Parameters is IParameters, ACL {
     uint256 public override aggregationRange = 3;
     uint256 public override withdrawReleasePeriod = 5;
     uint256 public override resetLockPenalty = 1;
-
+    // Note : maxAssetsPerStaker should be less than total no of jobs
```

Comment:
what happens if it isnt?

Reply:
If it isn't , the protocol doesn't remain efficient 
As there will be more chances of the same job assigned again.
code won't break but

Reply:
@abhishekvispute  maxAssetsPerStaker will always be less than no. of jobs like if we have 1000 jobs then there can be let's say 50 assets so maxAssets < no. of jobs always. So if a staker is assigned all the assets then also it will be less than the number of jobs right ?

Reply:
Hey @GauravJain9, Assets and Job are same in this context

Reply:
oh got it thanks @abhishekvispute